tests/task/test_training.py

```diff
@@ -19,15 +20,15 @@
 
 
 def validate_alignments(corpus_path, vocab_src_path, vocab_trg_path):
-    sp_src = spm.SentencePieceProcessor(model_file=vocab_src_path)
-    sp_trg = spm.SentencePieceProcessor(model_file=vocab_trg_path)
+    sp_src = spm.SentencePieceProcessor(model_file=vocab_src_path)  # type: ignore
```

Comment:
Same here, I'll stop commenting on the type suppressions, but the exclude list should be fixed instead.